.github/pull_request_template.md

```diff
@@ -1,3 +1,37 @@
+<!--
+
+Hello! Thank you for submitting a docs PR to terraform.io! Feel free to delete
+this message.
+
+- For advice or edits from a tech writer or education engineer,
+  please request review from the "hashicorp/terraform-education" GitHub team.
+
+- When updating screenshots of a web UI, please try to capture
+  the full width of the page, with the viewport size set to 1024px wide.
+
+- If you're a HashiCorp employee with permission to merge to this repo,
+  please get an approving review before merging your own PRs. (If you got
+  review approval on the private fork, that's fine too; just announce it in a
+  comment before merging!)
+
+- To learn more about how the website is built and deployed, how to preview your
+  changes, which content lives where, and more, check out the README.md in the
+  root of this repo.
+
+- When in doubt, ask in the #proj-terraform-docs channel.
```

Comment:
I don't think we should include this since it isn't available to the public. Alternately we could include it under the employee bullet.

Reply:
Yeah, I went back and forth on that. 🤔 Maybe under the employee bullet.